app/src/main/java/com/example/evently/ui/organizer/OwnEventsFragment.java

```diff
@@ -0,0 +1,28 @@
+package com.example.evently.ui.organizer;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.function.Consumer;
+
+import com.example.evently.data.model.Event;
+import com.example.evently.ui.common.EventsFragment;
+
+public class OwnEventsFragment extends EventsFragment {
```

Comment:
Similarly, your entrant fragment would be like `BrowseEventsFragment` and it would have to implement `initEvents` as "all active events" (can't do that until DB integration but you can have placeholders like you have in your PR)